utils/fatresize/Makefile

```diff
@@ -2,13 +2,13 @@ include $(TOPDIR)/rules.mk
 
 PKG_NAME:=fatresize
 PKG_VERSION:=1.1.0
-PKG_RELEASE:=1
```

Comment:
Hmm? Why bump?

Reply:
As `PKG_VERSION` is specified, the download script will still generate fatresize-1.1.0.tar.xz file.

I would suggest removing PKG_VERSION directly and resetting PKG_RELEASE to 1.

Reply:
@neheb
Because previous version is same, shouldn't release then be bumped?

@1715173329 

I'm ok, with all available resolutions. Newer commits just seemed important enough to provide update. You to decide together what solution should we proceed here, and then I'll make necessary changes to my PR, ok?

Reply:
Version is 1.1.0 + some patches that haven't yet drafter to a release.. So version probably should be there- or not?
So, which approach would be correct?

Reply:
from my point
```patch
-PKG_VERSION:=1.1.0
-PKG_RELEASE:=2
+PKG_RELEASE:=1
```

Reply:
@1715173329 
Changes pushed.

@neheb - what do you think of it?

Reply:
Looks correct